### PR TITLE
refractor our mock data generation, add resources to mock data

### PR DESCRIPTION
**Pull request by mat10tng**

### CODE REVIEW

File: scheme/data.py [hunk 1]
```diff
@@ -60,231 +59,264 @@ def genId():
 ]
 
 
+def any_time():
+    return int(time.time())
```
Comment (qdozaq) [inline scheme/data.py:63]: Looks like we need to multiply this by 1000.

other wise getting back `1589377870` which gives Fri Jan 02 1970  when run though `Date`

Comment (mat10tng) [reply scheme/data.py:63]: Date(1589377870)
"Wed May 13 2020 16:17:10 GMT+0200 (Central European Summer Time)"

File: scheme/data.json [hunk 2]
```diff
@@ -1,3385 +1,1864 @@
 {
   "organizations": {
-    "1": {
+    "f112d652ecf13dacd9c78c11e1e7f987": {
```
Comment (qdozaq) [inline scheme/data.json:3]: We probably want this to stay "1" for now right?

Comment (mat10tng) [reply scheme/data.json:3]: yeah,

**Pull request closed without merging**